org.jacoco.core.test/src/org/jacoco/core/runtime/RuntimeTestBase.java

```diff
@@ -38,7 +38,7 @@
 
 	private TestStorage storage;
 
-	abstract IRuntime createRuntime();
+	public abstract IRuntime createRuntime();
```

Comment:
No need to make this public when new runtime is in same package (see my other comment).